Library/SharedFunctions.swift

```diff
@@ -157,9 +157,20 @@ public func countdownProducer(to date: Date)
 }
 
 internal func is1PasswordButtonHidden(_ isHidden: Bool) -> Bool {
-  if #available(iOS 12, *) {
+  if AppEnvironment.current.isOSVersionAvailable(12.0) {
     return true
   } else {
     return isHidden
   }
 }
+
+public func ksr_isOSVersionAvailable(_ version: Double) -> Bool {
+  switch version {
+  case 12.0...:
```

Comment:
If we needed to test say `12.1` vs `12.1.1` then we'd just make this `switch` more specific, right?

Reply:
Yes, that is correct....as long as the order is correct and the higher version is declared before the open range operator..

<img width="1072" alt="screen shot 2018-12-10 at 3 18 27 pm" src="https://user-images.githubusercontent.com/387596/49759168-f02f7e00-fc8e-11e8-982d-2b8d745db92e.png">